streams/src/test/java/org/apache/kafka/streams/integration/IQv2StoreIntegrationTest.java

```diff
@@ -1350,7 +1355,7 @@ private static Properties streamsConfiguration(final boolean cache, final boolea
                                                    final String supplier, final String kind) {
         final String safeTestName =
             IQv2StoreIntegrationTest.class.getName() + "-" + cache + "-" + log + "-" + supplier
-                + "-" + kind;
+                + "-" + kind + "-" + RANDOM.nextInt();
```

Comment:
Allows us to iterate quickly on the tests without colliding on the state stores or waiting for the session timeout.